src/components/form/FAutocomplete.vue

```diff
@@ -282,6 +282,8 @@ function handleChange(event: Event): void {
 }
 
 function handleInput(e: InputEvent): void {
+  if (!(e instanceof InputEvent)) return;
```

Comment:
Of what type is it, then, and when is it triggered ?
Should we not prevent `FInput` itself to emit the `@input` event it this case ?

Reply:
`CustomEvent` emitted by maska!
Yes, I can move it to FInput directly!

Reply:
`CustomEvent` emitted by maska!
Yes, I can move it to FInput directly!